samples/ra-vpn-sfc/redirector-lb/ipv4subnetpool.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: cnfw.cisco.com/v1
+kind: IPv4SubnetPool
+metadata:
+  name: ravpnpool
+  namespace: sfcn-system 
+  annotations:
+    # enables AWS Route Table integration for this pool
+    aws.cnfw.cisco.com/type: "route-table"
+    # EP's node interface index that will be used as route target
+    aws.cnfw.cisco.com/interface-index: "3"
+    # AWS Route Table ID that will be synced with assigned subnets.
+    # This should be the table that includes your inside networks, usually named "Inside Subnets"
+    aws.cnfw.cisco.com/route-table-id: <your_route_table_id>
```

Comment:
this was enhanced in 2.0 to allow users to just add `inside` as the value. that way they don't have to lookup the ID themselves

Reply:
Done